Fix: Emails are converted to attachments in MS Exchange due encoded value in Content-Disposition header.

@issue https://github.com/magento/magento2/issues/25076

@package zendframework/zend-mail
@version >=2.10.0

--- src/Headers.php.org	2019-10-28 12:00:52.998605927 +0100
+++ src/Headers.php	2019-10-28 12:06:44.672407149 +0100
@@ -515,7 +515,10 @@
         }
 
         $current = $headers;
-        $current->setEncoding($encoding);
+        // PATCH
+        //$current->setEncoding($encoding);
+        $current->setEncoding($key === 'contentdisposition' && $current->getFieldValue(false) === 'inline' ? 'ASCII' : $encoding);
+        // ~PATCH
         $this->headers[$index] = $current;
         return $current;
     }
